docs/user-guide/odin_mcstas_helper.py

```diff
@@ -0,0 +1,343 @@
+# SPDX-License-Identifier: BSD-3-Clause
```

Comment:
Was this PR just about moving code around, or is a closer look required?

Reply:
Yes, it is just about moving code around : D...! 

> or is a closer look required?

So not really